### DISCUSSION TITLE
What's It Take To Get Api

### DISCUSSION THREAD

**Discussion by Shartair** [etsy/open-api#787] [Q&A]
I have applied 3 times for a api to use with my wordpress installation,and each time its never responded to. I am not building a app. 
but with out etsy api i can use my etsy sore in Wordpress without it . Could you please approve api so i can use my etsy store with Wordpress. I have to re apply because my last request just got removed with no response. ETSY STORE ON WORDPRESS DOES NOT WORK WITHOUT API.

**Comment by emergingdzns** [comment]
Hey @Shartair are you using some plugin? If so, maybe provide the name of that plugin so that the Etsy folks can look into it. Usually the apps get rejected because the application didn't include the URL to the website that would be using it or the description of what the app is supposed to do isn't clear enough. But there is supposed to be a reason shown. Hope that helps.
